scripts/complete_liquidity_provision.js

```diff
@@ -15,8 +15,9 @@ const { proceedAnyways } = require("./utils/user_interface_helpers")
 const { toErc20Units } = require("./utils/printing_tools")
 const { sleep } = require("./utils/js_helpers")
 const { verifyBracketsWellFormed } = require("./utils/verify_scripts")(web3, artifacts)
-
+const { DEFAULT_NUMBER_OF_SAFES } = require("./utils/constants")
```

Comment:
nit: could we call this `DEFAULT_NUM_SAFES`... I have this thing with putting "connector words" into variable names (i.e. `OF`)